gst/tensor_filter/tensor_filter.c

```diff
@@ -206,12 +206,10 @@ static gboolean gst_tensor_filter_stop (GstBaseTransform * trans);
  * @brief Open nn framework.
  */
 #define gst_tensor_filter_open_fw(filter) do { \
-      if (filter->prop.fw_opened == FALSE) { \
+      if (filter->prop.fw_opened == FALSE && filter->fw) { \
         if (filter->fw->open != NULL) {\
           if (filter->fw->open (filter, &filter->privateData) == 0) \
             filter->prop.fw_opened = TRUE; \
-          else \
-            filter->prop.fw_opened = FALSE; \
```

Comment:
Please explain why this line is removed.

Reply:
because of line 209, first condition is filter->prop.fw_opened == FALSE, so this is unnecessary.